app/models/revision.rb

```diff
@@ -74,4 +75,10 @@ def image_revisions_without_lead
   def assets
     image_revisions.flat_map(&:assets) + file_attachment_revisions.map(&:asset)
   end
+
+  def featured_attachments
+    file_attachment_revisions.sort_by do |attachment|
+      featured_attachment_ordering.find_index(attachment.featured_attachment_id).to_i
```

Comment:
Same point on making an informed choice of the consequences of items not in the list - may just need something in commit message.

Reply:
(I've also updated this commit to explain a bit more about the choice.)

Reply:
Great thanks